.travis.yml

```
@@ -4,22 +4,22 @@ jobs:
   include:
     - stage: "Tests & coverage"
       before_install:
-        - travis_retry npm install -g codecov
+        - travis_retry yarn add -g codecov
```

```suggestion
        - travis_retry yarn global add codecov
```
This syntax error is what's causing the current CI failures